packages/labs/task/src/task.ts

```diff
@@ -193,8 +197,8 @@ export class Task<T extends [...unknown[]] = any, R = any> {
       });
     }
     this.status = TaskStatus.PENDING;
+    this._value = this._intialValue;
```

Comment:
I thought we decided to the value stick around and not reset it at all?

Reply:
updated to not reset at all